hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java

```diff
@@ -994,6 +997,53 @@ public void testActiveCreateNonRecursiveDenyParallelReadOnAtomicDir() throws Exc
     Assert.assertTrue(fileSystem.exists(new Path("/hbase/dir/file")));
   }
 
+  @Test
+  public void testActiveCreateNonRecursiveNotDenyParallelReadOnAtomicDirIfLeaseConfigDisabled() throws Exception {
+    Assume.assumeTrue(
+        getFileSystem().getAbfsStore().getAbfsConfiguration().getPrefixMode()
```

Comment:
Can be getPrefixMode(fs) and can we reduce the length of the method name ?

Reply:
taken.